server/graphql/core/src/loader/vaccine_course_dose_by_vaccine_course.rs

```diff
@@ -29,6 +29,7 @@ impl Loader<String> for VaccineCourseDoseByVaccineCourseIdLoader {
         let mut map: HashMap<String, Vec<VaccineCourseDoseRow>> = HashMap::new();
 
         for dose in doses {
+            let dose = dose.vaccine_course_dose_row;

```

Comment:
Updated the `VaccineCourseDoseRepository` to return the `VaccineCourseDose` join/domain struct rather than just `VaccineCourseDoseRow`